### PR TITLE
LPS-105112 due the start of the server, the serviceContext is not properly initialized, and the creation of the ddmstructure is needed the user and group, we must insert it artificially on the serviceContext

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:lima

**Comment by AliciaGarciaGarcia** [comment]
ci:test:lima

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
Just started reviewing :)

File: modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/servlet/taglib/ui/LayoutCustomTagsScreenNavigationEntry.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Alicia Garcia
+ */
+@Component(
+	property = "screen.navigation.entry.order:Integer=3",
+	service = ScreenNavigationEntry.class
+)
+public class LayoutCustomTagsScreenNavigationEntry
```
Comment (adolfopa) [inline modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/servlet/taglib/ui/LayoutCustomTagsScreenNavigationEntry.java]: This class (and the accompanying JSP) should be in the `layout-seo-web` module.  We should try to keep as much SEO things in our modules as possible.

File: modules/apps/layout/layout-seo-service/service.xml [hunk 2]
```diff
@@ -33,6 +33,7 @@
 		<column localized="true" name="openGraphTitle" type="String" />
 		<column name="openGraphTitleEnabled" type="boolean" />
 		<column name="lastPublishDate" type="Date" />
+		<column name="DDMStorageId" type="long" />
```
Comment (adolfopa) [inline modules/apps/layout/layout-seo-service/service.xml]: Column names must begin with lowercase (e.g. "ddmStorageId")

File: modules/apps/layout/layout-seo-service/src/main/java/com/liferay/layout/seo/service/impl/LayoutSEOEntryLocalServiceImpl.java [hunk 3]
```diff
@@ -161,7 +170,42 @@ private LayoutSEOEntry _addLayoutSEOEntry(
...
+		Group companyGroup = _groupLocalService.getCompanyGroup(
+			group.getCompanyId());
+
+		DDMStructure ddmStructure = _ddmStructureLocalService.fetchStructure(
+			companyGroup.getGroupId(),
+			_classNameLocalService.getClassNameId(
+				LayoutSEOEntry.class.getName()),
+			"custom-opengraph-meta-tags");
+
+		DDMForm ddmForm = new DDMForm();
```
Comment (adolfopa) [inline modules/apps/layout/layout-seo-service/src/main/java/com/liferay/layout/seo/service/impl/LayoutSEOEntryLocalServiceImpl.java]: SF

**Pull request closed without merging**